frontend/public/kubevirt/components/vm-template/vm-template-detail.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { TEMPLATE_TYPE_LABEL } from 'kubevirt-web-ui-components';
+
+import { DetailsPage } from '../factory/okdfactory';
+import { breadcrumbsForOwnerRefs, navFactory, SectionHeading, ResourceSummary } from '../utils/okdutils';
+import { menuActions } from './menu-actions';
+import { kindForReference, LabelSelector } from '../../module/okdk8s';
+
+const DetailsForKind = kind => function DetailsForKind_({obj}) {
+  return <React.Fragment>
```

Comment:
no need for React.Fragment

Reply:
Fixed. This component was just copied from `default-resource.jsx` (is not exported).

The reason behind this is that vm template detail may be extended in the future.
I could also make a PR against the upstream to export this component.